regimes/mx/complements/fuel_complement.go

```diff
@@ -0,0 +1,58 @@
+// Package complements provides GOBL Complements for Mexican invoices
+package complements
```

Comment:
Not sure what's the best way of structuring complements. For the moment, I'm putting everything under a `complements` package within the `mx` path. Suggestions on how to better do this are welcome!

Reply:
I'm not sure... if you imagine a user who needs to set up complements for multiple countries, they'd have to deal with the namespace collisions of `complements` in multiple places. Could be awkward. I'd also think about the final schema URL, I was thinking of: `https://gobl.org/draft-0/regimes/mx/fuel-complement`

Reply:
Thanks, Sam! Agreed, there's no need for a `complements` package. Moved it to the main `mx` package in e178705. I also generated the schema file in the URL that you advised in d344e885f7a59d1097803aa2ac80db2cce8bdf13